ftml/src/render/html/builder.rs

```diff
@@ -30,6 +31,9 @@ macro_rules! tag_method {
     };
 }
 
+/// These are HTML tags which do not need a closing pair.
+const SOLO_HTML_TAGS: [&str; 3] = ["br", "br", "input"];
```

Comment:
Should the full list of self-closing tags be put in here? As these are only the ones that FTML is currently using - but common tags like the `<img/>` tag self-close, too.

Reply:
Oh, also, `br` is in there twice.

Reply:
Yeah I can do that.

Reply:
Also yes the second one was meant to be `hr`. I had replaced the `HashSet` with an array since it was faster and type-copied the name wrong.